cmd/snapshot.go

```diff
@@ -36,6 +37,11 @@ and symbol information.`,
 				Destination: &snapshotFlags.output,
 				Value:       "scip-snapshot",
 			},
+			&cli.StringFlag{
+				Name:        "project-root",
+				Usage:       "Override projet root in SCIP file",
```

Comment:
```suggestion
				Usage:       "Override project root in the SCIP file. For example, this can be helpful when the SCIP index was created inside a Docker image or created on another computer.",
```